### PR TITLE
Update cluster-setup to compile kn properly

### PR DESCRIPTION
**Pull request by rhuss**
Also, update hello plugin to cobra 1.0

### CODE REVIEW

File: plugins/hello/vendor/github.com/spf13/cobra/bash_completions.md [hunk 1]
```diff
@@ -56,7 +56,149 @@ func main() {
...
+	ArgAliases: argAliases
+}
+```
+
+The aliases are not shown to the user on tab completion, but they are accepted as valid nouns by
+the completion algorithm if entered manually, e.g. in:
+
+```bash
+# kubectl get rc [tab][tab]
+backend        frontend       database
```
Comment (mattmoor) [inline plugins/hello/vendor/github.com/spf13/cobra/bash_completions.md:108]: Remove trailing whitespace:
```suggestion
backend        frontend       database
```

Comment (mattmoor) [inline plugins/hello/vendor/github.com/spf13/cobra/bash_completions.md:108]: Remove trailing whitespace:
```suggestion
backend        frontend       database
```

Comment (mattmoor) [inline plugins/hello/vendor/github.com/spf13/cobra/bash_completions.md:108]: Remove trailing whitespace:
```suggestion
backend        frontend       database
```

Comment (mattmoor) [inline plugins/hello/vendor/github.com/spf13/cobra/bash_completions.md:108]: Remove trailing whitespace:
```suggestion
backend        frontend       database
```

Comment (mattmoor) [inline plugins/hello/vendor/github.com/spf13/cobra/bash_completions.md:108]: Remove trailing whitespace:
```suggestion
backend        frontend       database
```

**Comment by rhuss** [comment]
@daisy-ycguo I think we can merge that and then the issue on the non-vendor PR would be gone, too.

**Pull request merged**